docker/run_tests_in_docker.sh

```diff
@@ -62,63 +59,62 @@ if [ "$1" == "--shell" ]; then
            --env-file ./docker/config/test.env \
            --tty \
            --interactive \
+           --entrypoint="" \
            "${TESTIMAGE}" /bin/bash
+    exit $?
```

Comment:
This `exit` causes this script to break out early so we can dedent the `else` block.